
<html>
<head>
    <title>Single-Cell RNA Sequencing Data Analysis</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css">
    <script src='https://kit.fontawesome.com/a076d05399.js' crossorigin='anonymous'></script>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css">
    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;600&display=swap" rel="stylesheet">
    <!--Stylesheet-->
    <link rel="stylesheet" href="style.css">
    <script>
        document.addEventListener('DOMContentLoaded', function () {
            const toggler = document.getElementById('toggler');
            const menu = document.querySelector('.menu');
    
            document.addEventListener('click', function (event) {
                // Check if the toggler is checked and the click is outside the menu
                if (toggler.checked && !menu.contains(event.target) && event.target !== toggler) {
                    toggler.checked = false; // Uncheck the toggler to close the menu
                }
            });
    
            // Add an additional click event listener for clicks inside the menu
            menu.addEventListener('click', function (event) {
                // Check if the toggler is checked and the click is inside the menu
                if (toggler.checked && event.target !== toggler) {
                    toggler.checked = false; // Uncheck the toggler to close the menu
                }
            });
        });
    </script>
</head>
<style>
    @import url('https://fonts.googleapis.com/css2?family=Nunito:wght@200;300;400;600;700;800&amp;display=swap');

*,
*:before,
*:after {
  -moz-box-sizing: border-box;
  -webkit-box-sizing: border-box;
  box-sizing: border-box;
}

body {
  margin: 0;
}

.wk-desk-1 {
  width: 8.333333%;
}

.wk-desk-2 {
  width: 16.666667%;
}

.wk-desk-3 {
  width: 25%;
}

.wk-desk-4 {
  width: 33.333333%;
}

.wk-desk-5 {
  width: 41.666667%;
}

.wk-desk-6 {
  width: 50%;
}

.wk-desk-7 {
  width: 58.333333%;
}

.wk-desk-8 {
  width: 66.666667%;
}

.wk-desk-9 {
  width: 75%;
}

.wk-desk-10 {
  width: 83.333333%;
}

.wk-desk-11 {
  width: 91.666667%;
}

.wk-desk-12 {
  width: 100%;
}

@media (max-width: 1024px) {
  .wk-ipadp-1 {
    width: 8.333333%;
  }

  .wk-ipadp-2 {
    width: 16.666667%;
  }

  .wk-ipadp-3 {
    width: 25%;
  }

  .wk-ipadp-4 {
    width: 33.333333%;
  }

  .wk-ipadp-5 {
    width: 41.666667%;
  }

  .wk-ipadp-6 {
    width: 50%;
  }

  .wk-ipadp-7 {
    width: 58.333333%;
  }

  .wk-ipadp-8 {
    width: 66.666667%;
  }

  .wk-ipadp-9 {
    width: 75%;
  }

  .wk-ipadp-10 {
    width: 83.333333%;
  }

  .wk-ipadp-11 {
    width: 91.666667%;
  }

  .wk-ipadp-12 {
    width: 100%;
  }
}

@media (max-width: 768px) {
  .wk-tab-1 {
    width: 8.333333%;
  }

  .wk-tab-2 {
    width: 16.666667%;
  }

  .wk-tab-3 {
    width: 25%;
  }

  .wk-tab-4 {
    width: 33.333333%;
  }

  .wk-tab-5 {
    width: 41.666667%;
  }

  .wk-tab-6 {
    width: 50%;
  }

  .wk-tab-7 {
    width: 58.333333%;
  }

  .wk-tab-8 {
    width: 66.666667%;
  }

  .wk-tab-9 {
    width: 75%;
  }

  .wk-tab-10 {
    width: 83.333333%;
  }

  .wk-tab-11 {
    width: 91.666667%;
  }

  .wk-tab-12 {
    width: 100%;
  }
}

@media (max-width: 500px) {
  .wk-mobile-1 {
    width: 8.333333%;
  }

  .wk-mobile-2 {
    width: 16.666667%;
  }

  .wk-mobile-3 {
    width: 25%;
  }

  .wk-mobile-4 {
    width: 33.333333%;
  }

  .wk-mobile-5 {
    width: 41.666667%;
  }

  .wk-mobile-6 {
    width: 50%;
  }

  .wk-mobile-7 {
    width: 58.333333%;
  }

  .wk-mobile-8 {
    width: 66.666667%;
  }

  .wk-mobile-9 {
    width: 75%;
  }

  .wk-mobile-10 {
    width: 83.333333%;
  }

  .wk-mobile-11 {
    width: 91.666667%;
  }

  .wk-mobile-12 {
    width: 100%;
  }
}
    .navbar {
        position: fixed;
        background-image : linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0.1)), url(https://ukhsa.blog.gov.uk/wp-content/uploads/sites/33/2013/12/shutterstock_112250879-double-helix-dna.jpg);
        top: 0;
        width: 100%;
        background-color:whitesmoke;
        padding: 15px 0;
        z-index: 1000;
        display: flex;
        justify-content: space-between;
        align-items: center;
        background-size: cover; 
    }
    .navbar a {
        text-decoration: none;
        color: black	;
        }
    
    .navbar img{
        float:left;
        padding-left: 10px;
        width:max-content;
        height: 80px;
    }
    
    .navbar-columns {
        display: flex;
        list-style: none;   
        gap: 10px;
       
    }
    
    .navbar-column {
        margin: 0 20px;
        text-decoration: none;
        color:black;
        font-weight:bold;
        transition: color 0.6s ease;
        font-size:16px;
    }
    #toggler, label{
        display: none;
        }
    
        label i{
        color:black;
        align-items: right;
        padding-right: 30px ;
        }
        @media screen and (max-width:864px)
    {
            .menu {
            width: 100%;
            max-height: 0;
            height: 100vh; /* Set the height to the full height of the viewport */
            overflow: hidden;
            line-height: 40px;
            background-color: rgba(173, 216, 230, 0.9); /* Set a transparent background color */
            position: fixed;
            top: 0;
            left: 0;    
            z-index: 999; /* Set a higher z-index to appear above other elements */
            }
            
            .menu a {
            display: block;
            padding: 15px;
            text-align: center;
            color: black;
            font-size: 16px;
            transition: background-color 0.6s ease;
            }
            
            .menu a:hover {
            background-color: #999;
            }
            
            #toggler:checked~.menu {
            max-height: 100%;
            }
    
            .dropdown-content {
            display: none;
            position: absolute;
            background-color: #ebf3f8;
            min-width: max-content;
            border-radius: 5px;
            z-index: 1;
            top: 0;
            left: 100%; /* Position the dropdown content to the right of the dropdown button */
            }
    
            .dropdown:hover .dropdown-content {
            display: block;
            background-color: #7a8775;
            }
            
            label{
            display: inline-flex;
            cursor: pointer;
            }
    
            #toggler:checked~.menu{
            max-height: 100%;
            }
    
            .navbar-columns{
            flex-direction: column;
            align-items:center;
            padding:30px;
            }
    }
    
.navbar-column:hover {
    color:orangered; 
}
    .navbar-column:hover {
        color:orangered; 
    }
    
    .dropdown .dropbtn {
        font-size: 16px;  
        border: none;
        outline: none;
        color: white;
        padding: 10px 20px;
        background-color: inherit;
        font-family: inherit;
        margin: 0;
        border-radius: 5px;
        cursor: pointer ;
        position: relative;
      }
    
      .dropdown-content {
        display: none;
        position: absolute;
        background-color:#ebf3f8; /* Same color as the button */
        min-width: max-content;
        text-decoration: none;
        border-radius: 5px;
        z-index: 1;
      }
      
      .dropdown-content a {
        float: none;
        font-size: 90%;
        color: black;
        height:50px;
        padding: 8px 16px;
        text-decoration: none;
        display: block;
        text-align: left;
      }
      
      .dropdown-content a:hover {
        background-color: #999;
      }
      
      .dropdown:hover .dropdown-content {
        display: block;
        background-color: #7a8775;
      }

    .container {
        color: #151320;
            max-width: 1300px;
            margin: 0 auto;
            padding: 20px;
        
        }
        img {
    max-width: 400px;
    height:220px;
    float: right;
    display: block;
    border-radius: 20px 20px 0 0;
  }

  p {
            font-size: 12pt;
        }
        body {
            background-color:white ;
            font-family:'Poppins',sans-serif;
            margin: 0;
            padding: 0;
            
        }
        header {
            background-color:white;
            color:black;
            padding: 20px;
            text-align: center;
        }
        @media screen and (max-width: 768px) {
            .footer-columns {
                font-size: small;
                flex-direction: column;
                align-items: flex-start;
            }
        
            .footer-column {
                flex: auto;
                padding: 10px;
                text-align: center;
            }
        
            .social-icons {
                margin-top: 10px;
                text-align: center;
            }
           
        }
        
        .footer {
            width: 100%;
            background-color:#333;
            color:whitesmoke;
            padding: 20px 50px;
            font-weight:400;
        }
    
        .footer-columns {
            display: flex;
            justify-content: space-between;
            align-items: center;
        }
    
        .footer-column {
            flex: 1;
        }
    
        .footer-column p {
            margin: 1;
            
        }
    
        .social-icons {
            font-size: 30px;
        }
    
        .social-icons a {
            color: #fff;
            text-decoration:solid;
            margin-right: 20px;
        }
</style>
<body><br><br>  
        <div class="navbar">
            <img src="bioCOS-removebg-preview.png" alt="Logo">
        <label for="toggler">
            <i class="fa fa-bars"></i>
        </label>
    <input type="checkbox" id="toggler" name="">
    <div class="menu">    
        <div class="navbar-columns">
            <a href="../bioCOS HOME/index.html" class="navbar-column">HOME</a>
            <div class="dropdown"><strong><a href="../Insight_Analysis/main_analysis.html" style="text-decoration:none; color:black;" class="navbar-column" >ANALYSIS</a></strong>
            <div class="dropdown-content">
                <a href="../Insight_Analysis/d.html">DNA Analysis</a>
                <a href="../Insight_Analysis/r.html">RNA Analysis</a>
                <a href="../Insight_Analysis/scr.html">Single-Cell Analysis</a>
                <a href="../Insight_Analysis/epig.html">Epigenomic Analysis</a>
                <a href="../Insight_Analysis/p_m.html">Proteomic Analysis</a>
            </div></div>

            <div class="dropdown"><strong><a href="../R&D 2nd file/research.html" style="text-decoration:none; color:black;" class="navbar-column" >RESEARCH</a></strong>
                <div class="dropdown-content">
                    <a href="../R&D 2nd file/research_cancer1.html">CANCER</a>
                    <a href="../R&D 2nd file/research_genetics1.html">GENETICS</a>
                    <a href="../R&D 2nd file/research_nervous1.html">NERVOUS SYSTEM</a>
                    <a href="../R&D 2nd file/research_microbiology1.html">MICROBIOLOGY</a>
                    <a href="../R&D 2nd file/research_immunology1.html">IMMUNOLOGY</a>
                    <a href="../R&D 2nd file/research_drug1.html">DRUG DEVELOPMENT</a>
                </div></div>
            <a href="../reference page/index_final.html" class="navbar-column">REFERENCES</a>
            <a href="../aboutusnew/aboutus2.html" class="navbar-column">ABOUT US</a>
        </div>
    </div>
    </div>
    <br><br><br>

    <header>
        <h2>Scientific Research Center</h2>
        <p style="color:#151320 ;">Your Source for Cutting-Edge Research</p>
    </header>
    
    <div class="container">
        <h2 style="color:black">Quality Control and Preprocessing</h2>
        <div class="image">
            <img src="https://ars.els-cdn.com/content/image/1-s2.0-S0022283622001346-ga1.jpg" alt="Quality Control and Preprocessing">
<p>Like with any NGS data, the analysis of single-cell sequencing data starts with quality control and preprocessing.</p>
<ul>
    <li>Raw sequencing reads are quality-tested and metrics such as cell quality, accuracy, and diversity are generated.</li>
    <li>Reads are then aligned to an applicable reference genome or transcriptome, and additional metrics such as the number of cells, reads per cell, genes per cell, sequencing saturation, and fraction of mitochondrial transcripts are plotted and inspected.</li>
</ul>

<p>These QC metrics inform us about the total quality of the libraries and the usability of the samples and enable identifying and removing low-quality cells.</p>

<p>Further preprocessing is often carried out to remove unwanted signal, or noise, from certain downstream analyses. These include:</p>
<ol>
    <li>Imputation to estimate read counts for dropouts, or genes with zero transcripts due to technical, rather than biological, reasons,</li>
    <li>Normalization to remove biases due to e.g., differences in cell sizes, and</li>
    <li>Reducing the data to representative variables such as highly-variable genes or principal components.</li>
</ol>
<br><hr></div>
<h2 style="color:black">Cell Type Identification</h2>
<div class="image">
    <img src="https://blog.bioturing.com/wp-content/uploads/2022/03/1-s2.0-S2001037021000192-gr1.jpg" alt="Cell Type Identification">
<p>Identifying and characterizing cell types (and more refined cell states) is the most central part of most single-cell projects.</p>

<p>It all starts with identifying features (e.g., genes, proteins, accessible regions) that are specific to each cell cluster. These markers are defined by differential expression (DE) comparison of each cell cluster and the remaining ones, yielding DE statistics such as fold change and statistical significance.</p>

<p>The cluster markers can be visualized using scatter plots, violin plots, and heatmaps.</p>

<p>Markers are further annotated to biologically meaningful terms, such as biological processes, signaling pathways or a specific disease. Such analyses may rely either on over-representation analysis or gene set enrichment analysis, which both result in a list of enriched gene sets with relevant statistics and annotations.</p>

<p>Single-cell datasets are typically also integrated with publicly available data in order to exploit the cell-type information from already annotated datasets or cell atlases. This enables transferring cell labels into the analyzed dataset.</p>

<p>The transferred cell labels and identified markers and their annotations are used, together with prior information on cell-type/state markers, to identify the captured cell types.</p>
<br><br><hr></div>
<h2 style="color:black">Trajectory Analysis</h2>
<div class="image">
    <img src="https://bioconductor.org/books/3.14/OSCA.advanced/trajectory_files/figure-html/traj-princurve-tsne-nest-1.png" alt="Trajectory Analysis">
<p>In addition to characterizing distinct cellular identities, single-cell data lends itself to identifying continuums of gradual change in cell state, or trajectories. Uncovering such continuums is also called pseudotime analysis â€” while all cells are sampled at the same time point, individual cells may represent different stages in a temporal process such as differentiation.</p>

<p>De novo reconstruction of lineage differentiation and cell maturation trajectories allow exploring cellular dynamics, delineation of cell developmental lineages, and characterization of transition between cell states along a latent pseudotime dimension.</p>

<p>An ensemble of trajectory inference algorithms may be used for robust identification of root and terminal cellular states, branching points, and lineages. Single cells are ranked across deterministic or probabilistic lineages, and their ranking indicates their progression in a dynamic process of interest.</p>

<p>This type of analysis may also utilize the ratio of processed and unprocessed transcripts to infer whether a gene's expression is increasing or decreasing in a given cell. Combining this information from all quantified genes at a given state enables inferring the direction and pace of change in states. This is called RNA velocity analysis.</p>
<br><br><hr></div>
<h2 style="color:black">Integrative Single-Cell Analyses</h2>
<div class="image">
    <img src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41576-019-0093-7/MediaObjects/41576_2019_93_Fig1_HTML.png" alt="Integrative Single-Cell Analyses">
<p>Integrative single-cell analyses bring different datasets, including different data types and species together. This enables more accurate and detailed cell labeling and mechanistic insight into gene regulation in the studied system. Such analyses rely on common properties, or anchors, between the datasets, such as matched features (e.g., genes or homologues) or matched cells.</p>

<h3>Integrating Multiple Single-Cell RNA-Seq Datasets</h3>
<p>Perhaps the most common integration of single-cell datasets takes place between scRNA-seq datasets from different sources or technology platforms. Using genes as anchors, a successful integration removes the technical bias while retaining biological variance of the datasets.</p>

<p>Combining different scRNA-seq datasets is particularly helpful when there is a well-characterized public expression atlas available for a relevant tissue or organism.</p>

<h3>Integrating Single-Cell RNA-Seq and Epigenomics</h3>
<p>Integrating single-cell RNA-seq data with single-cell ATAC-seq or single-cell methylation data often relies on matched cells as anchors (when the measurements derive from the same cells as in, e.g., 10X Genomics Multiome technology).</p>

<p>Combining expression data with chromatin accessibility or methylation profiles enables more robust identification of cell types and allows for quantifying the effect of chromatin state to expression in individual cell types.</p>

<h3>Integrating Single-Cell RNA-Seq and Proteomics</h3>
<p>Since proteins, rather than transcripts, are key drivers of cellular functions, single-cell proteomics complements scRNA-seq experiments with more accurate estimates of cells functional states.</p>

<p>Single-cell proteomic profiling (CITE-seq, flow cytometry, mass cytometry, and mass spectrometry) comes in different degrees of throughput (number of proteins quantified) and can be targeted specifically to surface proteins, as in CITE-seq which involves a panel surface proteins quantified from cells with matched scRNA-seq reads.</p>

<p>Surface proteins are particularly useful in cell type identification, while the inclusion of cytosolic proteins enable better characterization of pathway and gene-regulatory activities.</p>

<h3>Cross-Species Integration</h3>
<p>Cross-species integrative analysis enables the identification of cell-type phylogenies that define the relationships of evolutionary and developmental mechanisms between different organisms. Shared homologues are used as anchors in cross-species integration.</p>

<p>This is particularly helpful when a disease/organ is better characterized on a single-cell resolution in an animal model than in human.</p>
<br><br><br><hr></div>
<h2 style="color:black">Ligand-Receptor Analysis</h2>
<div class="image">
    <img src="https://msraredon.github.io/Connectome/reference/figures/Big_Connectome.png" alt="Ligand-Receptor Analysis">
<p>Ligand-receptor (LR) analysis uncovers cell-cell interactions that coordinate homeostasis, development, and other system-level functions. Changes and dysfunction in such interactions may go unnoticed in an analysis limited to the internal state of individual cells or cell types.</p>

<p>Ligand-receptor analysis identifies and quantifies intercellular interactions based on the expression of known receptors and their ligands. The interactions may take place within or between tissues, and the strength of this interaction is compared between biological conditions of interest, such as patient groups, disease states, and treatments.</p>
<br><br><br><br><hr></div>
<h2 style="color:black">Spatial Transcriptomic Analysis</h2>
<div class="image">
    <img src="https://sites.dartmouth.edu/cqb/files/2020/11/Spatial-Transcriptomics.jpeg" alt="Spatial Transcriptomic Analysis">
<p>Spatially resolved single-cell transcriptomic assays couple expression data with the cell's positional context in a tissue or organ. This is particularly useful in the study of complex solid tissues, such as tumors and their microenvironment.</p>

<p>Spatial transcriptomic analysis involves cell/spot clustering in space, identification of spatially variable genes and resolving cell types in space.</p>

<p>Retaining the positional information of sequenced cells adds to the accuracy of identifying cell types and ligand-receptor interactions. It also enables spatial visualization of gene expression or chromatin accessibility (in the case of scATAC-seq) and integrating imaging-based data to the analysis.</p>

<p>Even in the case of lower-resolution assays, like 10X Visium, multimodal spatial analysis helps in correcting gene expression values and imputing dropout events.</p>
<br><br><hr></div></div>
    <div class="footer">
        <div class="footer-columns">
            <div class="footer-column">
                <p>BioCOS LIFE SCIENCES</p>
                <p>Bengaluru Area, India</p>
            </div>
            <div class="footer-column social-icons">
                <a href="https://www.researchgate.net/profile/Sanjeev-Kumar-215"><i class="fab fa-researchgate"></i></a>
                <a href="https://www.linkedin.com/company/biocos-life-sciences/?originalSubdomain=in"><i class="fab fa-linkedin-in"></i></a>
                <a href="https://www.contractresearchmap.com/providers/biocos-life-sciences"><i class="fab fa-connectdevelop"></i></a>
            </div>
            <div class="footer-column">
                <p>&copy; 2008 BioCOS LIFE SCIENCES</p>
                <p>Contact Us: biocos@biocosls.com</p>
            </div>
        </div>
    </div>
</body>
</html>
